charts/graphprotocol-node/values.yaml

```
@@ -8,7 +8,7 @@ image:
   repository: graphprotocol/graph-node
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: "v0.29.0"
```

```suggestion
  tag: ""
```
We are using .Chart.AppVersion for default value and providing these values to upgrade version without upgrading helm chart.
So we need to bump AppVarsion to v0.29.0 insted

---

Will add as follow-up PR